pages/index.tsx

```diff
@@ -21,9 +26,16 @@ const Home: NextPage = () => {
             <DateSelectors />
           </Grid.Col>
         </Grid>
-        <div style={{ paddingTop: '24px' }}>
-          <TestCaseEditor />
-        </div>
+        <Link href={'generateTestCases'}>
```

Comment:
best practice to not have camelCase link names since broswers can't really differentiate. I would suggest either renaming the page to something like `edit`??? or use kebab: `generate-test-cases`. This would involve renaming both the link and the file

Reply:
I don't _think_ this affects `next build` per se since the error message is quite cryptic, but you never know!

Reply:
Updated, but no luck on build :(